app/controllers/concerns/sufia/batch_uploads_controller_behavior.rb

```diff
@@ -5,7 +5,6 @@ module BatchUploadsControllerBehavior
     include CurationConcerns::CurationConcernController
 
     included do
-      layout "sufia-one-column"
```

Comment:
@mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.